app/controllers/citizens/transactions_controller.rb

```diff
@@ -31,7 +31,7 @@ def reset_selection
     end
 
     def set_selection
-      BankTransaction
+      bank_transactions
         .where(id: transaction_ids_to_select)
         .update_all(transaction_type_id: transaction_type.id)
```

Comment:
Would this be even better?
```ruby
bank_transactions
  .where(id: selected_transaction_ids)
  .update_all(transaction_type_id: transaction_type.id)
```

And we can get rid of the `transaction_ids_to_select` method